File: JavaScript_Courses/JavaScript_Basics/Preparation/03_Conditional_Statements_Advanced_Lab/12_Trade_Commissions.js
function tradeCommissions(input) {
    let city = input[0];
    let sales = Number(input[1]);

    if (city == "Sofia") {
        if (sales < 0) {
            console.log("error");
        } else if (sales <= 500) {
            console.log((sales * 0.05).toFixed(2));
        } else if (sales <= 1000) {
            console.log((sales * 0.07).toFixed(2));
        } else if (sales <= 10000) {
            console.log((sales * 0.08).toFixed(2));
        } else {
            console.log((sales * 0.12).toFixed(2));
        }
    } else if (city == "Varna") {
        if (sales < 0) {
            console.log("error");
        } else if (sales <= 500) {
            console.log((sales * 0.045).toFixed(2));
        } else if (sales <= 1000) {
            console.log((sales * 0.075).toFixed(2));
        } else if (sales <= 10000) {
            console.log((sales * 0.10).toFixed(2));
        } else {
            console.log((sales * 0.13).toFixed(2));
        }
    } else if (city == "Plovdiv") {
        if (sales < 0) {
            console.log("error");
        } else if (sales <= 500) {
            console.log((sales * 0.055).toFixed(2));
        } else if (sales <= 1000) {
            console.log((sales * 0.08).toFixed(2));
        } else if (sales <= 10000) {
            console.log((sales * 0.12).toFixed(2));
        } else {
            console.log((sales * 0.145).toFixed(2));
        }
    } else {
        console.log("error");
    }
}

tradeCommissions(["Sofia", "1500"]);

File: Python_Courses/Python_Basics/Preparation/03_Conditional_Statements_Advanced_Lab/12_Trade_Commissions.py
city = input()
sales = float(input())

trade_commissions = 0

if city == "Sofia":
    if 0 <= sales <= 500:
        trade_commissions = sales * 0.05
    elif 500 < sales <= 1_000:
        trade_commissions = sales * 0.07
    elif 1_000 < sales <= 10_000:
        trade_commissions = sales * 0.08
    elif sales > 10_000:
        trade_commissions = sales * 0.12
elif city == "Varna":
    if 0 <= sales <= 500:
        trade_commissions = sales * 0.045
    elif 500 < sales <= 1_000:
        trade_commissions = sales * 0.075
    elif 1_000 < sales <= 10_000:
        trade_commissions = sales * 0.10
    elif sales > 10_000:
        trade_commissions = sales * 0.13
elif city == "Plovdiv":
    if 0 <= sales <= 500:
        trade_commissions = sales * 0.055
    elif 500 < sales <= 1_000:
        trade_commissions = sales * 0.08
    elif 1_000 < sales <= 10_000:
        trade_commissions = sales * 0.12
    elif sales > 10_000:
        trade_commissions = sales * 0.145

if trade_commissions:
    print(f"{trade_commissions :.2f}")
else:
    print("error")
